budgetApp1/budgetApp1/UserViewModel.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+class UserViewModel {
+    
+    let authenticationRepository: AuthenticationRepositoryType
+    
+    var userData = [UserEntity]()
```

Comment:
var userData: [UserEntity]
The assigning can be done in the init.
Doesn't make sense to instantiate authenticationRepository in the init and then userData not in the init